data/companies.yml

```diff
@@ -602,7 +602,10 @@ users:
   - name: "GoTo Financial"
     logo: "/logos/gotofinancial.svg"
     url: "https://gotofinancial.com/"
-
+  - name: "Crystal Business Funding"
+    logo: "/logos/2lA8NvZqXk5hbjTObOtH37C4kAV.svg"
```

Comment:
```suggestion
    logo: "/logos/2lA8NvZqXk5hbjTObOtH37C4kAV.svg"
```

Please name this similar to `CrystalBusinessFunding.svg` and remove the other images you are adding. The `2lA8N...` appears to be the desired photo, but the others need to be dropped and this renamed to something more understandable. Happy to approve once this is done.

Reply:
I don't see the other photo's 
![IMG-20240826-WA0035](https://github.com/user-attachments/assets/f485995c-1659-48ef-9a16-acc34347920e)

Reply:
![Screenshot_219](https://github.com/user-attachments/assets/204446d0-4c12-4534-b9b5-67ef32528abb)